rust/src/types/mod.rs

```diff
@@ -1,13 +1,19 @@
 mod amount;
 mod code_reader;
+mod execution_context;
 mod memory;
+#[cfg(test)]
+mod mock_execution_message;
 mod opcode;
 mod stack;
 mod tx_context;
 
 pub use amount::u256;
 pub use code_reader::{CodeReader, GetOpcodeError};
+pub use execution_context::*;
 pub use memory::Memory;
+#[cfg(test)]
+pub use mock_execution_message::{MockExecutionMessage, DEFAULT_INIT_GAS};
```

Comment:
I guess there will be some day we have to reconsider include habits. In the meanwhile we forward symbols for visibility. I don't think we need to forward this one.